### PR TITLE
pkp/pkp-lib#2606: pt_PT locale updated (ojs base locale + plugins)

### PR DESCRIPTION
**Pull request by Duozhasht**
Greetings once again,

some more files on the base ojs project. If you wish to receive the plugins, that are represented as a submodule/upstream just tell me so. I will personally upload them as soon as possible.

Regards,
Tiago

### CODE REVIEW

File: plugins/generic/backup/locale/ojs-bck2/plugins/generic/backup/locale/pt_PT/locale.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mtub) [inline plugins/generic/backup/locale/ojs-bck2/plugins/generic/backup/locale/pt_PT/locale.xml]: this file probably shouldn't be here, right?

File: plugins/generic/recommendBySimilarity/locale/ojs-bck2/plugins/generic/recommendBySimilarity/locale/pt_PT/locale.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mtub) [inline plugins/generic/recommendBySimilarity/locale/ojs-bck2/plugins/generic/recommendBySimilarity/locale/pt_PT/locale.xml]: this file probably shouldn't be here, right?

File: plugins/paymethod/paypal/locale/pt_PT/locale.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  * plugins/paymethod/paypal/locale/pt_PT/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2006-2009 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the pt_BR (Português (Brasil)) locale.
+  -->
+ 
+<locale name="pt_BR" full_name="Português do Brasil">
```
Comment (mtub) [inline plugins/paymethod/paypal/locale/pt_PT/locale.xml]: change to pt_PT, Portugal etc.

File: plugins/generic/backup/locale/ojs-bck2/plugins/generic/backup/locale/pt_PT/locale.xml (hunk 1 above)
Comment (Duozhasht) [inline plugins/generic/backup/locale/ojs-bck2/plugins/generic/backup/locale/pt_PT/locale.xml]: Yep my bad, it was a `cp --parents` done wrong, and I didn't noticed.

File: plugins/generic/recommendBySimilarity/locale/ojs-bck2/plugins/generic/recommendBySimilarity/locale/pt_PT/locale.xml (hunk 2 above)
Comment (Duozhasht) [inline plugins/generic/recommendBySimilarity/locale/ojs-bck2/plugins/generic/recommendBySimilarity/locale/pt_PT/locale.xml]: same as above, fixed

File: plugins/paymethod/paypal/locale/pt_PT/locale.xml (hunk 3 above)
Comment (Duozhasht) [inline plugins/paymethod/paypal/locale/pt_PT/locale.xml]: Solved aswell, did a `grep -rnw './' -e 'Brasil' | egrep pt_PT`: found two more files
Files:
plugins/importexport/users/locale/pt_PT/locale.xml
plugins/block/developedBy/locale/pt_PT/locale.xml

Also had the same error. br files were my starting point

**Comment by mtub** [comment]
This is for pkp/pkp-lib#2606 - we also got another pt_PT translation, so we will have to figure out how to best merge the two.

**Pull request closed without merging**